Documentation/RelNotes/2.11.0.txt

```diff
@@ -5,7 +5,7 @@ Backward compatibility notes.
 
  * An empty string used as a pathspec element has always meant
    'everything matches', but it is too easy to write a script that
-   finds a path to remove in $path and run 'git rm "$paht"' by
+   finds a path to remove in $path and run 'git rm "$path"' by
```

Comment:
This is a typo all right, but it is an _intentional_ one: the whole point of the explanation is to give the reader an example why `git rm` might be called with an empty argument (and the example _is_ that the user had a typo in their command).

Reply:
Reverted.